entrypoint.sh

```diff
@@ -87,6 +87,13 @@ esac
 
 
 API_VERSION=($(python -c "from optimade import __api_version__; versions = [__api_version__.split('-')[0].split('+')[0].split('.')[0], '.'.join(__api_version__.split('-')[0].split('+')[0].split('.')[:2]), '.'.join(__api_version__.split('-')[0].split('+')[0].split('.')[:3])]; print(' '.join(versions))"))
+PACKAGE_VERSION=($(python -c "from optimade import __version__; print(__version__.replace('.', ' '))"))
+
+if [ ${PACKAGE_VERSION[0]} -eq 0 ] && [ ${PACKAGE_VERSION[1]} -lt 10 ]; then
+    echo "Incompatible validator version requested ${INPUT_VALIDATOR_VERSION}, please use >=0.10."
+    exit 1
+fi
+
```

Comment:
To at least fail a bit faster, this should be moved up to right after having installed the `optimade` package.

Also, it's fine to keep in the original checks, although this is cleaner, of course.